src/index.html

```diff
@@ -2,11 +2,434 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header" id="home">
+      <a href="#home" class="header__home header__home--margin">
+        <img
+          src="images/logo.png"
+          alt="moyo_icon"
+        >
+      </a>
+
+      <nav class="nav" >
+        <div class="nav__height">
+          <a
+            href="#Apple"
+            class="nav__link nav__link--margin is-active"
+          >
+            Apple
+          </a>
+
+          <a
+            href="#Samsung"
+            class="nav__link nav__link--margin"
+          >
+            Samsung
+          </a>
+
+          <a
+            href="#Smartphones"
+            class="nav__link nav__link--margin"
+          >
+            Smartphones
+          </a>
+
+          <a
+            href="#Laptops_&_Computers"
+            class="nav__link nav__link--margin"
+            data-qa="nav-hover"
+          >
+            Laptops&nbsp;&&nbsp;computers
+          </a>
+
+          <a
+            href="#Gadgets"
+            class="nav__link nav__link--margin"
+          >
+            Gadgets
+          </a>
+
+          <a
+            href="#Tablets"
+            class="nav__link nav__link--margin"
+          >
+            Tablets
+          </a>
+
+          <a
+            href="#Photo"
+            class="nav__link nav__link--margin"
+          >
+            Photo
+          </a>
+
+          <a
+            href="#Video"
+            class="nav__link nav__link--margin"
+          >
+            Video
+          </a>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container main__container--margin">
+        <div class="card card--margin-right" data-qa="card">
```

Comment:
```suggestion
        <div class="card main__card" data-qa="card">
```

Reply:
this is what is called BEM mixes fix it everywhere

Reply:
done